cmds/tui.c

```diff
@@ -1106,10 +1116,60 @@ static void print_graph_empty(void)
 	if (list_empty(&graph_output_fields))
 		return;
 
-	list_for_each_entry(field, &graph_output_fields, list)
+	list_for_each_entry(field, &graph_output_fields, list) {
 		printw("%*s", field->length + FIELD_SPACE, "");
+		field_width += field->length + FIELD_SPACE;
+	}
 
 	printw(FIELD_SEP);
+	field_width += FIELD_SPACE;
+}
+
+static bool is_boxchar(unsigned char p)
+{
+	/*
+	 * simply check whether the given character is a box character or not.
+	 *   ├ = 0xe2949c, │ = 0xe29482, └ = 0xe29494,
+	 *   ─ = 0xe29480, ▶ = 0xe296b6
+	 */
+	if (p >= (unsigned char)0x80 && p <= (unsigned char)0xb6)
+		return true;
+	else
+		return false;
+}
```

Comment:
I don't understand the logic.

Reply:
Please refer to the link below.
- https://docs.python.org/3/library/curses.ascii.html#curses.ascii.ismeta

I think I should modify this only to check if it's above `0x80`.

Reply:
The following python code explains it as well.

https://github.com/python/cpython/blob/283de2b9c18e38c9a573526d6c398ade7dd6f8e9/Lib/curses/ascii.py#L69

https://github.com/python/cpython/blob/283de2b9c18e38c9a573526d6c398ade7dd6f8e9/Python/bltinmodule.c#L1637-L1683